mantidproject/mantid
# Frequency Domain Analysis GUI

This is the first version (0.0) of the frequency domain analysis GUI for the moun group. This early version just calculates a FFT from some data. 

**To test:**
1. To test load multiple data sets into the muon analysis GUI (including different groups from the same run) and make a not of what you have imported. 

2. Launch the Frequency Domain Analysis GUI and check that the two drop down menus contain the data that you loaded into muon analysis (it will not include data ending in "_Raw" or the MuonAnalysis and MuonAnalysisGrouped workspaces). 

3. Play with the options and make some FFT's. Check the results by comparing the output (MuonFFT) with the output from using the FFT algorithm. 



Fixes #20144. 

**Release Notes** 

add to the release notes
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
